site/_utils/addCollectionByDirectory.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file Creates a collection for a directory under a specific locale
+ * while using the same path in the default locale as a fallback.
+ * For example used to show localized (or original) blog articles
+ * alongside untranslated ones.
+ */
+
+const path = require('path');
+const {filterOutDrafts} = require('./drafts');
+
+const defaultLocale = 'en';
+
+function addCollectionByDirectory(config, locale, dir) {
+  config.addCollection(`${dir}-${locale}`, collections => {
```

Comment:
I think this function should be defined in the _collections folder: https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/_collections and then the addCollection call can stay in the main eleventy file.